Tips-for-common-editors.md

```diff
@@ -0,0 +1,141 @@
+      {
+        "python.linting.pylintEnabled": true,
+        "python.linting.pylintPath": "../oppia_tools/pylint-1.9.5/bin/pylint"
+      }
+      ```
+
+    * Choose the correct Python interpreter from the blue bar at the bottom:
+
+      ![Changing the Python interpreter](images/commonEditors/vscodePythonInterpreter.png)
+
+  * JavaScript and TypeScript support are built-in to Code, but we can add the `dbaeumer.vscode-eslint` extension to add ESLint support. Then whenever there is a lint error in your code, you'll see a red squiggly line. If you hover over that line, you'll see a pop-up describing the problem:
+
+    ![ESLint error](images.commonEditors/vscodeEslint.png)
+
+* You can right-click on code elements (e.g. functions or variables) to find where they are defined and used.
+
+  ![Options from right-clicking code](images/commonEditors/vscodeRightClick.png)
+
+* To filter the file tree, first click on the file tree window. Then start typing. Your text will appear in the upper right corner, and matches will be highlighted.
+
+  ![Filtering file tree](images/commonEditors/vscodeFilterHighlight.png)
+
+  If you hover over the text that appears and click the icon with three horizontal lines, only matching files will be shown.
+
+  ![Hiding non-matches when filtering file tree](images/commonEditors/vscodeFilterHide.png)
+
+To learn more, check out the [Code documentation](https://code.visualstudio.com/docs).
+
+## Sublime
+
+To start editing Oppia code:
+
+1. Open the `oppia/` directory (e.g. using CMD+O on MacOS or CTRL+O on Linux or Windows).
+
+2. From the file tree on the left side, select a file to edit. It opens in the center. On the right is a minimap, which shows the file contents in small font so you can see more of the file at once.
+
+   ![Sublime with a file open for editing](images/commonEditors/sublimeEditor.png)
+
+3. To edit the file, just start typing!
+
+Tips:
+
+* [Package Control](http://www.sublimelinter.com/en/stable/) is the *de facto* package manager for Sublime. You can install it by selecting "Install Package Control" from the "Tools" menu. Then you can install packages by opening the command pallete (CMD-SHIFT-P on MacOS or CTRL-SHIFT-P on Windows) and typing `Package Control: Install Package`. We recommend the following packages:
+
+  * [`SublimeLinter`](https://packagecontrol.io/packages/SublimeLinter) is a base linter package. To add support for pylint and eslint, install the following packages as well:
+
+    * [`SublimeLinter-eslint`](https://packagecontrol.io/packages/SublimeLinter-eslint)
+
+    * [`SublimeLinter-pylint`](https://packagecontrol.io/packages/SublimeLinter-pylint)
+
+    To configure them, open the SublimeLinter settings from the menu: Preferences > Package Settings > SublimeLinter > Settings. Add these lines to the file that opens (should be mostly blank to start), replacing `PARDIR` with the absolute path to the folder containing both the `oppia_tools` and `oppia` directories:
```

Comment:
Perhaps {{PARDIR}} so it stands out a bit more. Also perhaps consider {{PARENT_DIR}} since this is generally aimed at newcomers.

Reply:
done